lib/oli_web/live/delivery/student/learn_live.ex

```diff
@@ -2490,14 +2505,21 @@ defmodule OliWeb.Delivery.Student.LearnLive do
         contained_pages_due_dates(
           section_or_subsection,
           student_end_date_exceptions_per_resource_id,
-          show_completed_pages
+          show_completed_pages,
+          ctx
         )
 
       _ ->
         []
     end)
   end
 
+  defp to_localized_date(datetime, ctx) do
```

Comment:
Maybe we can use the existing `OliWeb.Common.FormatDateTime.maybe_localized_datetime/2` function?